js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -89,6 +89,11 @@ var KeyboardNavigationController = core.ViewController.inherit({
         }
     },
 
+    _addTabIndexToElement: function($element) {
```

Comment:
I think that _addTabIndexToElement is not a good name. Change it to _applyTabIndexToElement.